src/Route.php

```diff
@@ -358,4 +377,14 @@ public function getOrder(): int
     {
         return $this->order;
     }
+
+    /**
+     * Get middleware status
+     *
+     * @return bool
+     */
+    public function useMiddleWare(): bool
```

Comment:
Be consistent with other methods: getMiddleWare

Reply:
done 👍